### DISCUSSION TITLE
Is this currently supported OCR with Windows APP SDK 1.6 （stable）?

### DISCUSSION THREAD

**Discussion by tjinjin95** [microsoft/WindowsAppSDK#4803] [General]
### Describe the bug

Describe the bug
ref: https://learn.microsoft.com/en-us/windows/ai/apis/text-recognition

I remember the documentation mentioned that Windows APP SDK 1.6 （stable）would support NGAI, such as OCR in the ref, but I didn’t see the relevant libraries when I actually used it. Is this currently supported?

![Image](https://github.com/user-attachments/assets/0b32455c-9559-444a-af03-442d4f49da3c)

### Steps to reproduce the bug

1.Create a WinUI3 app by VS2022
2.Nuget windows app sdk 1.6(Latest stable 1.6.240923002)
3.using Microsoft.Windows.Vision; using Microsoft.Windows.Imaging; -> (X)

### Expected behavior

OCR API can be found and work well.

### Screenshots

![Image](https://github.com/user-attachments/assets/f3fb9a69-3b01-4cce-9acc-170b9d1222bd)

### NuGet package version

Windows App SDK 1.6.1: 1.6.240923002

### Windows version

Windows 11 version 22H2 (22621, 2022 Update)

**Comment by castorix** [comment]
I didn't find _Microsoft.Windows.Vision_ either, in any DLL or .winmd or _microsoft-ui-xaml-winui3-release-1.6-stable_ source
(I had tested _Windows.Media.Ocr_ but it is not perfect...)

**Comment by hez2010** [comment]
I believe those APIs are only included in the experimental version of WASDK and don't exist in the stable version.

**Comment by aubakir** [comment]
> I believe those APIs are only included in the experimental version of WASDK and don't exist in the stable version.

Version 1.6 Experimental (1.6.0-experimental2)
This is the latest release of the experimental channel.

To download, retarget your WinAppSDK NuGet version to 1.6.240701003-experimental2.

 Note

Phi Silica and OCR APIs are not included in this release. These will be coming in a future 1.6 release.
